js/components/conceptset/included.js

```diff
@@ -55,8 +55,8 @@ define([
 				conceptSetStore: this.conceptSetStore,
 			});
 			commonUtils.clearConceptsSelectionState(this.includedConcepts);
-    }
-	
+			this.conceptSetStore.loadingIncluded(false);
```

Comment:
Little confused here: adding the concepts tot he concept set store will signal a reload which should handle clearing the `loadingIncluded` flag (once all the async requests are complete).  But this looks like it will set the `loadingIncluded` to true and then false almost immediately due to the async calls?

Reply:
You are right. We do not need to place loader here at all, Because if/when we apply changes from https://github.com/OHDSI/Atlas/pull/2460/files - there wiil be no async calls here to wait for. So i suggest to just remove this.conceptSetStore.loadingIncluded(true); from line 50.

Reply:
In any case, if we need to put a loader on the page, we rather not do it in this function, but where async calls are. Which is already there: js/components/conceptset/ConceptSetStore.js:158